kale/pipeline/domain_adapter.py

```diff
@@ -47,35 +47,35 @@ def set_requires_grad(model, requires_grad=True):
         param.requires_grad = requires_grad
 
 
-def get_aggregated_metrics(metric_name_list, metric_outputs):
-    """Get a dictionary of the mean metric values (to log) from metric names and their values"""
-    metric_dict = {}
-    for metric_name in metric_name_list:
-        metric_dim = len(metric_outputs[0][metric_name].shape)
-        if metric_dim == 0:
-            metric_value = torch.stack([x[metric_name] for x in metric_outputs]).mean()
-        else:
-            metric_value = torch.cat([x[metric_name] for x in metric_outputs]).double().mean()
-        metric_dict[metric_name] = metric_value.item()
-    return metric_dict
-
-
-def get_aggregated_metrics_from_dict(input_metric_dict):
-    """Get a dictionary of the mean metric values (to log) from a dictionary of metric values"""
-    metric_dict = {}
-    for metric_name, metric_value in input_metric_dict.items():
-        metric_dim = len(metric_value.shape)
-        if metric_dim == 0:
-            metric_dict[metric_name] = metric_value
-        else:
-            metric_dict[metric_name] = metric_value.double().mean()
-    return metric_dict
-
-
-# multi-GPUs: mandatory to convert float values into tensors
-def get_metrics_from_parameter_dict(parameter_dict, device):
-    """Get a key-value pair from the hyperparameter dictionary"""
-    return {k: torch.tensor(v, device=device) for k, v in parameter_dict.items()}
+# def get_aggregated_metrics(metric_name_list, metric_outputs):
+#     """Get a dictionary of the mean metric values (to log) from metric names and their values"""
+#     metric_dict = {}
+#     for metric_name in metric_name_list:
+#         metric_dim = len(metric_outputs[0][metric_name].shape)
+#         if metric_dim == 0:
+#             metric_value = torch.stack([x[metric_name] for x in metric_outputs]).mean()
+#         else:
+#             metric_value = torch.cat([x[metric_name] for x in metric_outputs]).double().mean()
+#         metric_dict[metric_name] = metric_value.item()
+#     return metric_dict
+
+
+# def get_aggregated_metrics_from_dict(input_metric_dict):
+#     """Get a dictionary of the mean metric values (to log) from a dictionary of metric values"""
+#     metric_dict = {}
+#     for metric_name, metric_value in input_metric_dict.items():
+#         metric_dim = len(metric_value.shape)
+#         if metric_dim == 0:
+#             metric_dict[metric_name] = metric_value
+#         else:
+#             metric_dict[metric_name] = metric_value.double().mean()
+#     return metric_dict
+
+
+# # multi-GPUs: mandatory to convert float values into tensors
+# def get_metrics_from_parameter_dict(parameter_dict, device):
+#     """Get a key-value pair from the hyperparameter dictionary"""
+#     return {k: torch.tensor(v, device=device) for k, v in parameter_dict.items()}
```

Comment:
Is the above no longer needed? Shall we delete rather than comment them off then? Any benefits of keeping them?